service/docs/source/install.rst

```diff
@@ -1,115 +1,86 @@
-
-In addition to these packages that are built each time a tracked
-branch is updated, the download repositories also provide
-``python3-dasbus`` version 1.6.  This version of dasbus is required by
-``python3-geopmdpy``, and is not provided by the supported Linux
-distributions.
-
-Installing the ``geopm-service`` package will also install the
-``libgeopmd2``, ``python3-geopmdpy`` and ``python3-dasbus`` dependency
-packages.  The ``geopm-service-devel`` package must be explicitly
-installed if it is required by the user.
-
-
-Level Zero Prerequisite
------------------------
-
-Before installing the GEOPM Service packages, the ``level-zero``
-package must be installed.  This package enables GEOPM's support for
-GPU metrics and controls.  The ``level-zero`` package is not currently
-provided upstream by the Linux distributions that GEOPM supports.
-Please follow the
-`Intel(R) GPGPU software package installation guide for Linux <https://dgpu-docs.intel.com/installation-guides/index.html>`__
-to install the ``level-zero`` package on your system.  The linked
-guide shows how to add the OS appropriate package management
-repository to your system.  Once the repository has been added, use
-``yum`` or ``zypper`` to install the ``level-zero`` package.  Only the
-``level-zero`` package is required, there is no GEOPM requirement to
-install any of the other packages mentioned in the guides linked
-above.  Installing the ``level-zero-devel`` package will enable the
-user to build GEOPM from source with GPU support if that is desired.
+Each time specified branches hosted on GitHub are updated, continuous
+integration generates packages for numerous Linux distributions. This
+process uses GitHub Actions, combined with the OpenSUSE Build Service
+(OBS). Here are the packages curated by OBS:
+
+- ``geopm-service``: Sets up and enables the geopm systemd service.
+- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
+- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
+- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
+
+Besides these routinely curated packages, the download repositories also grant
+access to ``python3-dasbus`` version 1.6. This version is a prerequisite
+for ``python3-geopmdpy``, but not included by the Linux distributions that
+back us.
+
+Installation of ``geopm-service`` will involve the dependency packages:
+``libgeopmd2``, ``python3-geopmdpy`` and ``python3-dasbus``. However, explicit
+installation is obligatory for anyone seeking to use ``geopm-service-devel``.
+
+
+Prerequisites for Level Zero
+----------------------------
+
+Before you install the GEOPM Service packages, ensure you have the
+``level-zero`` package. This package supports GEOPM for GPU metrics and
+controls. Currently, none of the Linux distributions supporting GEOPM
+provide this package upstream.
+
```

Comment:
Level Zero dependent GEOPM packages are provided in the supplementary repo, but the default repo does not support Level Zero.  This should be clarified.